test/api.ts

```diff
@@ -133,6 +133,19 @@ test('#sendEmail: override body: success', (t) => {
   t.is(req.message.transactional_message_id, 1);
 });
 
+test('#getCustomersByEmail: searching for a customer email (default)', (t) => {
+  sinon.stub(t.context.client.request, 'get');
+
+  const email = 'hello@world.com';
+  t.context.client.getCustomersByEmail(email);
+  t.truthy((t.context.client.request.get as SinonStub).calledWith(`${RegionUS.apiUrl}/customers?email=${email}`));
+})
+
+test('#getCustomersByEmail: should throw error when email is empty', (t) => {
```

Comment:
Another test should be added for cases where `null`/`undefined` or a non-string object are used for non-typescript clients

Reply:
done!